django/contrib/admin/static/admin/js/collapse.js

```diff
@@ -30,23 +30,26 @@
                 h2.appendChild(document.createTextNode(')'));
             }
         }
-        // Add toggle to anchor tag
-        var toggles = document.querySelectorAll('fieldset.collapse a.collapse-toggle');
+        // Add toggle to hide/show anchor tag
         var toggleFunc = function(ev) {
-            ev.preventDefault();
-            var fieldset = closestElem(this, 'fieldset');
-            if (fieldset.classList.contains('collapsed')) {
-                // Show
-                this.textContent = gettext('Hide');
-                fieldset.classList.remove('collapsed');
-            } else {
-                // Hide
-                this.textContent = gettext('Show');
-                fieldset.classList.add('collapsed');
+            if (ev.target.matches('.collapse-toggle')) {
+                ev.preventDefault();
+                ev.stopPropagation();
+                var fieldset = closestElem(ev.target, 'fieldset');
+                if (fieldset.classList.contains('collapsed')) {
+                    // Show
+                    ev.target.textContent = gettext('Hide');
+                    fieldset.classList.remove('collapsed');
+                } else {
+                    // Hide
+                    ev.target.textContent = gettext('Show');
+                    fieldset.classList.add('collapsed');
+                }
             }
         };
-        for (i = 0; i < toggles.length; i++) {
-            toggles[i].addEventListener('click', toggleFunc);
+        var inlineDivs = document.querySelectorAll('fieldset.module');
```

Comment:
I widened this from `div.inline-group` to account for `admin_views.tests.SeleniumTests.test_collapsible_fieldset`

Reply:
Yeah. It did. 🙂